libs/pages/elewa/social-impact/src/lib/social-impact-foote-section/social-impact-foote-section.component.html

```diff
@@ -0,0 +1,8 @@
+<elewa-group-footer>
+    <div footer class="footer-content">
+        <h4>do you want to be part of our mission?</h4>
+        <div class="mission">
+          <elewa-group-button-with-animation [message]="'Become an investor'" [mode]="'dark'" [action]="'/'"></elewa-group-button-with-animation>
+        </div>
+      </div>
```

Comment:
The files are wrongly named - foote

Also you don't need a component for the footer on the page modules since it's already added as part of the page layout.